### ISSUE TITLE
Connecting from the server different from 3.0.2 to 3.1.0

### ISSUE THREAD

**Issue by ckir** [primus/primus#378]
The following code works fine when using 3.0.2 but on 3.1.0
the new socket(masterUrl) returns a string (something like '9952db7a-fa01-435f-a29e-04940ffd2e4c').

What needs to be done ?
```js
var Primus = require('primus');
var PrimusCallbacks = require('primus-callbacks');

var socket = Primus.createSocket({
    transformer: 'socket.io',
    parser: 'JSON',
    requestTimeout: 30000,
    plugin: {
        callbacks: PrimusCallbacks
    }
});

var client = new socket(masterUrl);
var killWorkersTimeout = 0;
var self = this;

console.log(require('util').inspect(client, { showHidden: true, depth: null }));
```

**Comment by lpinca** [comment]
Should be fixed in 3.1.1.
More info on the issue here: https://github.com/primus/primus/pull/379#issuecomment-111076240.

**Comment by lpinca** [comment]
Forgot to say thank you for reporting!
